data/moves.js

```diff
@@ -14550,6 +14550,9 @@ exports.BattleMovedex = {
 			if (myItem) {
 				target.setItem(myItem);
 				this.add('-item', target, myItem, '[from] move: Trick');
+			}else {
+				target.setItem(myItem);
+				this.add('-enditem', target, 'Air Balloon');
```

Comment:
Well, this line is probably the right idea, except for the whole 'Air Balloon' part, and it should probably be `[silent]` Do we have `[silent]` support for `-enditem`?